### PR TITLE
test: add zkp issuance scenario

### PR DESCRIPTION
**Pull request by amagyar-iohk**
# Overview

Fixes #<issue number>

### My PR contains...
* [ ] No code changes (changes to documentation, CI, metadata, etc.)
* [ ] Bug fixes (non-breaking change which fixes an issue)
* [ ] Improvements (misc. changes to existing features)
* [ ] Features (non-breaking change which adds functionality)

### My changes...
* [ ] are breaking changes
* [ ] are not breaking changes
* [ ] If yes to above: I have updated the documentation accordingly

### Documentation
* [ ] My changes do not require a change to the project documentation
* [ ] My changes require a change to the project documentation
* [ ] If yes to above: I have updated the documentation accordingly

### Tests
* [ ] My changes can not or do not need to be tested
* [ ] My changes can and should be tested by unit and/or integration tests
* [ ] If yes to above: I have added tests to cover my changes
* [ ] If yes to above: I have taken care to cover edge cases in my tests

### CODE REVIEW

**Comment by atala-dev** [comment]
### Code Coverage
|Total Project Coverage|26.42%|
|:-|:-:|

File: tests/end-to-end/src/test/kotlin/io/iohk/atala/prism/configuration/Environment.kt [hunk 1]
```diff
@@ -159,15 +160,72 @@ object Environment {
...
+            tags = listOf("automation"),
+            type = "AnoncredSchemaV1",
+            version = "2.0.0",
+            schema = anoncredSchema
+        )
+
+        val schemaCreationResponse = RestAssured.given()
+            .body(credentialSchemaInput)
+            .post("/schema-registry/schemas")
+            .thenReturn()
```
Comment (antonbaliasnikov) [inline tests/end-to-end/src/test/kotlin/io/iohk/atala/prism/configuration/Environment.kt:208]: It's nice to check the status here, to make sure schema was created.

File: tests/end-to-end/src/test/kotlin/io/iohk/atala/prism/utils/Keys.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package io.iohk.atala.prism.utils
+
+object Keys {
+    val CONNECTION_ID = "connectionId"
+
+}
```
Comment (antonbaliasnikov) [inline tests/end-to-end/src/test/kotlin/io/iohk/atala/prism/utils/Keys.kt:6]: Endline here.

**Review by antonbaliasnikov — APPROVED** [review]
LGTM!
